src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -59,6 +57,31 @@ const HotspotDetails = () => {
   const { t } = useTranslation()
   const selectedHotspots = hotspotsToFeatures([hotspot])
   const { purpleMain, greenOnline } = useColors()
+  const { showActionSheetWithOptions } = useActionSheet()
+  const [showSettings, setShowSettings] = useState(false)
+
+  const onMoreMenuSelected = () => {
+    const options: string[] = [
+      ...t('hotspot_details.more', { returnObjects: true }),
```

Comment:
returning an array here, and then making assumptions below based on the index feels a little implicit. would it be worth having this be a bit more explicit at the cost of being more verbose?

Reply:
you could maybe create an object with labels and handler functions. would remove the need for the switch below, potentially

Reply:
yeah, I'm with you. It does feel a bit smelly.

Reply:
Your suggestion is similar to what I did for the more page. I'll go with that.